assets/airports/rjbb.json

```diff
@@ -0,0 +1,2732 @@
+		"NANKO": ["N34d36m38.70",	"E135d21m21.00"],
+		"OGURA": ["N34d47m58.70",	"E135d34m24.90"],
+		"OYE"  : ["N34d45m01.00",	"E133d50m06.00"],
+		"SIGAK": ["N34d53m07.70",	"E136d02m52.70"],
+		"TSC"  : ["N34d07m47.00",	"E134d36m31.00"],
+		"UMAJI": ["N33d45m00.50",	"E134d09m00.60"],
+		"BRAVE": ["N33d25m50.00",	"E133d33m30.00"],
+		"FUE":   ["N32d40m07.00",	"E128d49m36.00"],
+		"KARIN": ["N33d40m40.00",	"E134d11m50.00"],
+		"KISEI": ["N33d28m04.00",	"E135d19m44.00"],
+		"OOITA": ["N33d13m13.00",	"E131d42m12.00"],
+		"RANDY": ["N34d06m35.00",	"E133d53m51.00"],
+		"SALTY": ["N33d51m10.00",	"E132d55m31.00"],
+		"STORK": ["N32d56m48.00",	"E133d14m35.00"],
+		"SUC":   ["N32d45m21.00",	"E132d59m48.00"],
+		"TAPOP": ["N32d40m01.00",	"E136d07m57.00"]
+	},
+    "runways": [{
+        "name": ["06L", "24R"],
+        "name_offset": [
+            [0, 0], [0, 0]
+        ],
+        "end": [
+            ["N34d25m42.86", "E135d12m22.33", "122.5ft"],
+            ["N34d27m04.58", "E135d14m24.08", "122.6ft"]
+        ],
+        "length": 4.000,
+        "ils": [true, true]
+    }, {
+        "name": ["06R", "24L"],
+        "name_offset": [
+            [0, 0], [0, 0]
+        ],
+        "end": [
+            ["N34d25m02.56", "E135d13m45.40", "122.8ft"],
+            ["N34d26m14.04", "E135d15m31.92", "123.0ft"]
+        ],
+        "length": 3.500,
+        "ils": [true, true]
+    }],
+    "sids": {
+        "MAIKO6": {
+            "icao": "MAIKO6",
+            "name": "Maiko Six",
+            "exitPoints": {
+                "MAIKO": [
+                    ["MAIKO", "A50+"]
+                ]
+            },
+            "rwy": {
+                "06L": ["KNE", ["MAIKO", "A50+"]],
+                "06R": ["KNE", ["KCE", "A25+"], ["MAIKO", "A50+"]],
+                "24L": [],
+                "24R": []
+            },
+            "draw": [
+                ["KNE", "KCE", "MAIKO"],
+                ["KNE", "MAIKO"]
+            ]
+        },
+        "FERRY5": {
+            "icao": "FERRY5",
+            "name": "Ferry Five",
+            "exitPoints": {
+                "MIYAZU": [
+                    ["KAWAT", "A80+"], "CHIZU", "YME"
+                ],
+                "KIBI": [
+                    ["KAWAT", "A80+"], "OYE"
+                ],
+                "KAGAWA": [
+                    ["KAWAT", "A80+"], "KTE"
+                ]
+            },
+            "rwy": {
+                "24L": ["MAIKO"],
+                "24R": ["MAIKO"]
+            },
+            "draw": [
+                ["MAIKO", "KAWAT"],
+                ["KAWAT", "CHIZU", "YME*"],
+                ["KAWAT", "OYE*"],
+                ["KAWAT", "KTE*"]
+            ]
+        },
+        "TOMOH1": {
+            "icao": "TOMOH1",
+            "name": "Tomoh One",
+            "exitPoints": {
+                "KOCHI_S": [
+					"UMAJI", "KRE"
+				],
+                "KUSHIMOTO": [
+					["GOBOH", "A60+"], "KEC"
+				]
+            },
```

Comment:
Due to the need to type the name of the exit point when assigning aircraft a new SID, please use the proper ICAO fix/navaid identifier for all entry and exit points:

```
"entryPoints": {
    "KRE": [
        "KRE"
    ],
    "KEC": [
        "KEC"
    ]
}
```